Source/Applications/Wave Demo Apps/UpdateWAVMetaData/Program.cs

```diff
@@ -109,12 +94,12 @@ static int Main(string[] args)
                     if (Convert.ToInt32(connection.ExecuteScalar(database.ParameterizedQueryString("SELECT COUNT(*) FROM Device WHERE Acronym = {0}", "acronym"), acronym)) == 0)
                     {
                         parameterizedQuery = database.ParameterizedQueryString("INSERT INTO Device(NodeID, Acronym, Name, ProtocolID, FramesPerSecond, " +
-                            "MeasurementReportingInterval, ConnectionString, Enabled) VALUES(" + nodeIDQueryString + ", {0}, {1}, {2}, {3}, {4}, {5}, {6})",
+                            "MeasurementReportingInterval, ConnectionString, Enabled) VALUES({0}, {1}, {2}, {3}, {4}, {5}, {6}, {7})",
                             "acronym", "name", "protocolID", "framesPerSecond", "measurementReportingInterval",
-                            "connectionString", "enabled");
+                            "connectionString", "enabled", "nodeID");
 
                         // Insert new device record
-                        connection.ExecuteNonQuery(parameterizedQuery, acronym, name, protocolID, sourceWave.SampleRate, 1000000, $"wavFileName={FilePath.GetAbsolutePath(sourceFileName)}; connectOnDemand=true; outputSourceIDs={acronym}; memoryCache={useMemoryCache}", database.Bool(true));
+                        connection.ExecuteNonQuery(parameterizedQuery, acronym, name, protocolID, sourceWave.SampleRate, 1000000, $"wavFileName={FilePath.GetAbsolutePath(sourceFileName)}; connectOnDemand=true; outputSourceIDs={acronym}; memoryCache={useMemoryCache}", database.Bool(true), database.Guid(nodeID)); memoryCache ={useMemoryCache}", database.Bool(true), nodeID);
```

Comment:
This doesn't work, because `NodeID` is the first column in the `INSERT` list. If you absolutely must put these fields at the end of the parameters list, then you need to also modify the `INSERT` list to put `NodeID` at the end. However, I think it would make more sense to go with my original suggestion and just put the `"nodeID"` and `database.Guid(nodeID)` parameters at the beginning of their respective parameter lists.

Reply:
GitHub is obnoxious. In case it helps, know that I had selected lines 96-102 before making this comment.